pkgs/top-level/aliases.nix

```
@@ -486,6 +486,7 @@ mapAliases {
   autoconf213 = throw "'autoconf213' has been removed in favor of 'autoconf'"; # Added 2025-07-21
   autoconf264 = throw "'autoconf264' has been removed in favor of 'autoconf'"; # Added 2025-07-21
   automake111x = throw "'automake111x' has been removed in favor of 'automake'"; # Added 2025-07-21
+  autopanosiftc = throw "'autopanosiftc' has been removed, as it is unmaintained upstream"; # Added 2025-10-07
```

Nit, i like to give info about since when something was unmaintained. 2009 is much worse than 2024, for example.
```suggestion
  autopanosiftc = throw "'autopanosiftc' has been removed, as it is unmaintained upstream since <year>"; # Added 2025-10-07
```